onnxruntime/contrib_ops/cpu/transformers/subgraph_base.cc

```diff
@@ -49,6 +50,13 @@ Subgraph::Subgraph(
   for (int i = 0; i < num_subgraph_outputs; ++i) {
     subgraph_output_names.push_back(subgraph_outputs[i]->Name());
   }
+
+  for (const auto& node : subgraph.Nodes()) {
+    if (node.OpType() == "DecoderMaskedMultiheadAttention") {
+      has_decoder_masked_multihead_attention_ = true;
+      break;
```

Comment:
Assume seeing one node of `DecoderMaskedMultiheadAttention` means all Attention nodes are `DecoderMaskedMultiheadAttention` in the graph - we will assume no mix of Attention kernels